resources/public/api/conf/1.0/testdata/reinstate-account.md

```diff
@@ -38,35 +38,60 @@
             </td>
         </tr>
         <tr>
-            <td><p>Request for an account that is open or active</p><p class="code--block">lisaManagerReferenceNumber: <a href="https://test-developer.service.hmrc.gov.uk/api-documentation/docs/api/service/lisa-api/1.0#testing-the-api">Use your test user profile</a><br>accountId: 8765432100</p></td>
+            <td><p>Request for an account that is open or active</p><p class="code--block">lisaManagerReferenceNumber: <a href="https://test-developer.service.hmrc.gov.uk/api-documentation/docs/api/service/lisa-api/1.0#testing-the-api">Use your test user profile</a><br>accountId: 0000000905</p></td>
                 <td><p> empty payload </p>
```

Comment:
Two things:
1. Missing a closing `</td>`
2. Please just leave it empty, don't specify 'empty payload'

Same thing applies in other areas of this file.

Reply:
(on line 42)